tests/parsers/unchained_labs_lunatic/testdata/spectrum_measurement.json

```diff
@@ -0,0 +1,13341 @@
+{
+    "$asm.manifest": "http://purl.allotrope.org/manifests/plate-reader/BENCHLING/2023/09/plate-reader.manifest",
+    "plate reader aggregate document": {
+        "plate reader document": [
+            {
+                "measurement aggregate document": {
+                    "measurement time": "2024-03-09T14:37:50+00:00",
+                    "plate well count": {
+                        "value": 96,
+                        "unit": "#"
+                    },
+                    "measurement document": [
+                        {
+                            "measurement identifier": "UNCHAINED_LABS_LUNATIC_TEST_ID_0",
+                            "device control aggregate document": {
+                                "device control document": [
+                                    {
+                                        "device type": "plate reader",
+                                        "detection type": "Absorbance",
+                                        "detector wavelength setting": {
+                                            "value": 230.0,
+                                            "unit": "nm"
+                                        },
+                                        "electronic absorbance reference wavelength setting": {
+                                            "value": 340.0,
+                                            "unit": "nm"
+                                        },
+                                        "custom information document": {
+                                            "path length mode": "single",
+                                            "pump": "0",
+                                            "electronic absorbance reference absorbance": 0.16
+                                        }
+                                    }
+                                ]
+                            },
+                            "sample document": {
+                                "sample identifier": "sample_A1",
+                                "location identifier": "A1",
+                                "well plate identifier": "Plate 1",
+                                "custom information document": {
+                                    "path length": 10.0
+                                }
+                            },
```

Comment:
One comment on placement of the `electronic absorbance reference absorbance` value. Would you be able to add as a sibling to `absorbance`. Otherwise ASM LGTM 👍